Sources/core/objects/OnMapBase.h

```diff
@@ -147,6 +147,7 @@ class IOnMapBase: public IDraw
     virtual void UpdatePassable() {}
     virtual IdPtr<ITurf> GetTurf();
     virtual void SetTurf(IdPtr<ITurf> turf);
+    virtual IdPtr<CubeTile> GetTile();
```

Comment:
1. It should be `GetRoot` instead `GetTile` (because it returns the root of the owner hierarchy, and it is not always tile).
2. It should not be virtual. It is a helper function which based on `GetOwner` function, and subclasses should not override it.